lte/protos/session_manager.proto

```diff
@@ -328,6 +328,7 @@ message ChargingCredit {
   FinalAction final_action = 5;
   GrantedUnits granted_units = 6;
   RedirectServer redirect_server = 7;
+  repeated string restrict_rules = 8;
```

Comment:
This proto was not defined properly. So lets take advantage now to do it right too. We should have the same structure as we mentioned before grouping all the final actions elements. 

```
message ChargingCredit {
  enum UnitType {
      BYTES = 0;
      SECONDS = 1;
  }
  UnitType type = 2;
  uint32 validity_time = 3; // seconds

  FinalUnitIndication final_unit_indication = 8
}

message FinalUnitIndication {
  enum FinalAction {
    TERMINATE = 0;
    REDIRECT = 1;
    RESTRICT_ACCESS = 2;
  }
  bool is_final = 1;    // <- You need this because protobuf doesn't have null
  FinalAction final_action = 2;
  GrantedUnits granted_units = 3;
  RedirectServer redirect_server = 4;
}
```


And that change will have impact on sessiond too

Reply:
This is outside the scope of this PR. Changing this will require modifying all sessiond code !!
Feel free to create a task for such change which should be tracked separately.